### PR TITLE
Use correct notation for multiple denominator units

### PR DESCRIPTION
**Pull request by Goodwine**
See: https://physics.nist.gov/cuu/Units/checklist.html

Tests: https://github.com/sass/sass-spec/pull/2027

### CODE REVIEW

**Comment by nex3** [comment]
I'm not entirely convinced this is better. We can't realistically match the SI standards here—I don't think users would really understand `px*em⁻¹*rad⁻¹*s⁻¹` even if we could rely on Unicode characters rendering well everywhere. `px*em/rad/s` is explicitly called out as *not* recommended. We could do `px*em/(rad*s)`, which does seem to be one of the allowed SI formats and is more clear to a naive reader.

**Comment by Goodwine** [comment]
Oops, I only skimmed through the doc and didn't actually read it lol

I like using parentheses in this case, I'll update the PR

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 1.79.6
```
Comment (nex3) [inline CHANGELOG.md:1]: We actually haven't released 1.79.5 yet, so you might as well put this in there too.

**Review by nex3 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Goodwine) [reply CHANGELOG.md:1]: oks, I will do that then :)

**Pull request merged**